apps/web/src/features/shared/textarea-autocomplete/_index.scss

```
@@ -1,5 +1,31 @@
+// because the @/features/shared barrel side-effect-imported the comment composer.
+// That re-export was dropped (#850/#852), which stripped the styles (notably
+// `width: 100%`) from /submit and squashed the input column.
+.the-editor {
+  @include themify(night) {
+    @apply text-silver;
+    @apply bg-dark-200;
+  }
+  @include themify(day) {
+    @apply bg-light-200;
+  }
+  border-width: 0px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)